api/src/user/dto/update-user.dto.ts

```diff
@@ -0,0 +1,37 @@
+import { ApiProperty } from '@nestjs/swagger';
+import { IsNotEmpty, IsOptional, IsUUID, ValidateIf } from 'class-validator';
+
+export class UpdateUserDto {
```

Comment:
The basic user info is part of key cloak token, I think there is no scope of updating it in our database, again KeyCloak is the owner of that data. Our scope is to store user specific location information only.

Reply:
We want to update the location field though